ingestion/src/metadata/utils/dbt_config.py

```diff
@@ -117,15 +129,18 @@ def _(config: DbtHttpConfig):
             dbt_catalog = requests.get(  # pylint: disable=missing-timeout
                 config.dbtCatalogHttpPath
             )
-        return (
-            json.loads(dbt_catalog.text) if dbt_catalog else None,
-            json.loads(dbt_manifest.text),
-            json.loads(dbt_run_results.text) if dbt_run_results else None,
+        if not dbt_manifest:
+            raise DBTConfigException("Menifest file not found in file server")
```

Comment:
also breaking the workflow if menifest file is not found in case of file server/ s3/gcs since its required to process dbt info

Reply:
good catch